Author: Michael R. Crusoe <crusoe@debian.org>
Description: drop snappy feature
Forwarded: not-needed

Currently it is not part of librocksdb-sys in Debian, and the main anticipated user
of rust-rocksdb (sourmash) doesn't need it.
--- rocksdb.orig/Cargo.toml
+++ rocksdb/Cargo.toml
@@ -67,7 +67,6 @@
 [features]
 bzip2 = ["librocksdb-sys/bzip2"]
 default = [
-    "snappy",
     "lz4",
     "zstd",
     "zlib",
@@ -79,7 +78,6 @@
 multi-threaded-cf = []
 rtti = ["librocksdb-sys/rtti"]
 serde1 = ["serde"]
-snappy = ["librocksdb-sys/snappy"]
 valgrind = []
 zlib = ["librocksdb-sys/zlib"]
 zstd = ["librocksdb-sys/zstd"]
--- rocksdb.orig/tests/test_rocksdb_options.rs
+++ rocksdb/tests/test_rocksdb_options.rs
@@ -223,11 +222,6 @@
 }
 
 #[test]
-fn test_snappy_compression() {
-    test_compression_type(DBCompressionType::Snappy);
-}
-
-#[test]
 fn test_zlib_compression() {
     test_compression_type(DBCompressionType::Zlib);
 }
